srcpkgs/openssh/patches/sshd_config.patch

```diff
@@ -0,0 +1,37 @@
+--- a/sshd_config
++++ b/sshd_config
+@@ -9,6 +9,7 @@
+ # OpenSSH is to specify options with their default value where
+ # possible, but leave them commented.  Uncommented options override the
+ # default value.
++Include /etc/ssh/sshd_config.d/*.conf
```

Comment:
```suggestion

# To avoid conflicts with the packaged configuration, specify custom options
# in drop-in files under /etc/ssh/sshd_config.d to override any defaults or
# options set below.
+Include /etc/ssh/sshd_config.d/*.conf
```